tests/src/Context/ContentTypeContext.php

```diff
@@ -0,0 +1,203 @@
+<?php
+/**
+ * @file
+ * Contains \Drupal\nexteuropa\Context\FieldContext.
+ */
+
+namespace Drupal\nexteuropa\Context;
+
+use Behat\Behat\Context\Context;
+use Behat\Gherkin\Node\TableNode;
+
+/**
+ * Context with content type management.
+ */
+class ContentTypeContext implements Context {
+
+  /**
+   * The list of node type before a scenario starts.
+   *
+   * @var array
+   */
+  protected $defaultNodeTypes = array();
+
+  /**
+   * Remember the list of node type.
+   *
+   * @BeforeScenario
+   */
+  public function rememberCurrentNodeTypes() {
+    foreach (node_type_get_types() as $result) {
+      $this->defaultNodeTypes[$result->type] = $result->type;
+    }
+  }
+
+  /**
+   * Removes any node types created after the last list node type remembered.
+   *
+   * @AfterScenario
+   */
+  public function resetNodeTypes() {
```

Comment:
With this function, we can remove the following code (file FeatureContext.php - line 653)

    // Delete 'community' node type.
    _node_types_build(TRUE);
    node_type_delete('community');
    field_purge_batch(1);